CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Unreleased
+==========
+* Permit `text-1.2.5.0`.
```

Comment:
Actually I wonder if we should mention it in the changelog :thinking: 
I think the most proper way to release this change is to make a new revision on Hackage without updating a new version (that's what revisions are for). So maybe we should create `1.7.2 (rev1)` entry?

Reply:
Like this?

And then just upload the same `1.7.2` version on Hackage again?

Reply:
> And then just upload the same `1.7.2` version on Hackage again?

It's impossible to _upload_ it again, but it's possible to create a new revision.

![image](https://user-images.githubusercontent.com/2556409/128836453-7006989e-e394-469f-b450-74eef243d8f9.png)